connectivity/service/src/main/java/org/eclipse/ditto/connectivity/service/messaging/tunnel/SshTunnelActor.java

```diff
@@ -177,7 +177,8 @@ private void handleConnectResult(final ConnectFuture connectFuture) throws IOExc
             logger.debug("SSH session connected successfully.");
             sshSession = connectFuture.getClientSession();
             sshSession.addSessionListener(new TunnelSessionListener(getSelf(), logger));
-            sshSession.addChannelListener(new TunnelChannelListener(getSelf(), logger));
+            sshSession.addChannelListener(new TunnelChannelListener(getSelf(),
+                    this.connection.getSpecificConfig().getOrDefault("initialWindowSize","0"), logger));
```

Comment:
Maybe we should call it e.g. "initialSshChannelWindowSize" to be more precise what is affected by the config property.

Reply:
Good idea. I will change the name.